extern/sector-storage/stores/index.go

```diff
@@ -296,7 +296,31 @@ func (i *Index) StorageFindSector(ctx context.Context, s abi.SectorID, ft Sector
 	}
 
 	if allowFetch {
+		spaceReq, err := ft.SealSpaceUse(spt)
+		if err != nil {
+			return nil, xerrors.Errorf("estimating required space: %w", err)
+		}
+
 		for id, st := range i.stores {
+			if !st.info.CanSeal {
```

Comment:
So ideally this function would take `pathType PathType` and check this based on that, but looking at current usages of this function it shouldn't break anything (we should also use it for `spaceReq`)